Automattic/simplenote-electron
# Use md5 node module

### Fix
This uses a node module that only does md5 not a host of other hashing functions. This is only used in the sharing modal to pull down a users Gravatar

### Test
1. Go to https://app.simplenote.com/new
2. Click share icon in top right add a user to share with
3. In Chrome dev tools get the link to the image
4. Build this branch
5. Open app
6. Click share
7. Enter email
8. Get image link
9. Compare hash from production, this branch, and another md5 hash generator
10. They should all match
 
### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in d3adb3ef with:

- md5 hashing module was replaced

Closes: #1294
